src/OpcPlcServer.cs

```diff
@@ -164,24 +169,39 @@
     /// <summary>
     /// Start web server to host pn.json.
     /// </summary>
-    private void StartWebServer(IHost host)
+    private void StartWebServer(string[] args)
     {
         try
-        {
-            host.Start();
-
-            if (Config.ShowPublisherConfigJsonIp)
-            {
-                Logger.LogInformation("Web server started: {pnJsonUri}", $"http://{GetIpAddress()}:{Config.WebServerPort}/{Config.PnJson}");
-            }
-            else if (Config.ShowPublisherConfigJsonPh)
-            {
-                Logger.LogInformation("Web server started: {pnJsonUri}", $"http://{Config.OpcUa.Hostname}:{Config.WebServerPort}/{Config.PnJson}");
-            }
```

Comment:
We still need to show this information. Did you check if the pn.json can still be retrieved?

Reply:
restored web application hosting pn.json file